File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {


        int x1=x;
        int reverse=0;
        int n=0;
        while(x1>0){
        n=x1%10;
        reverse=reverse*10+n;
        x1=x1/10;
      
            
        }
        if(reverse==x){
            return true;
        }
        return false;
    }
}

File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        
        int x1=0;
        int x2=x;
        long x3=0;
        while(x2>0){
       
       x3=x3*10+x2%10;
      x2/=10;
        }
        if(x3==x){
            return true;

        }
        return false;
    }
};